doc/config_rsmtool.rst.inc

```diff
@@ -281,6 +281,13 @@ The values for a key can be string, integer, float, or boolean depending on what
 hyperparameter expects. Note that if this option is specified with the
 :ref:`built-in linear regression models <builtin_models>`, it will simply be ignored.
 
+.. _skll_grid_search_jobs:
+
+skll_grid_search_jobs *(Optional)*
+""""""""""""""""""""""""""""""""""
+Number of folds to run in parallel when using skll grid search. Defaults to 1 (no parallelization). For very large data
```

Comment:
Just a minor nitpick. It would be nice to distinguish the string `skill` with ticks to indicate it is the name of a library.

```suggestion
Number of folds to run in parallel when using ``skll`` grid search. Defaults to 1 (no parallelization). For very large data
```